help_to_heat/templates/frontdoor/feedback.html

```diff
@@ -12,72 +12,103 @@
   {% endif %}
 
 <h1 class="govuk-heading-l">{{_("Help us improve the service")}}</h1>
-<form id="feedback" method="post" novalidate>
+<form id="feedback" method="post">
     {{csrf_input}}
 
+    {% if errors %}
+    <div class="govuk-error-summary" data-module="govuk-error-summary">
+      <div role="alert">
+        <h2 class="govuk-error-summary__title">
+          {{_("Please answer at least one question before submitting feedback")}}
+        </h2>
+      </div>
+    </div>
+    {% endif %}
```

Comment:
This is rendered conditionally and is adapted from the error summary div in `macros.html`